test/lib/analyzer/image-inspector.spec.ts

```diff
@@ -85,6 +86,17 @@ describe("extractImageDetails", () => {
     expect(imageName).toEqual(expected.imageName);
     expect(tag).toEqual(expected.tag);
   });
+  it("test unprovided image", async () => {
```

Comment:
Looks like the pattern for test naming begins "should eg line 112", wdyt about aligning these?

Reply:
Aligned, however note that in the other tests "should produce the expected state" is not a very clear message. Could change it here if you find it like a good fit.